allennlp/models/simple_tagger.py

```diff
@@ -62,12 +58,13 @@ def forward(self,  # type: ignore
         ----------
         tokens : Dict[str, torch.LongTensor], required
             The output of TextField.as_array() which should typically be passed directly to a
-            ``TokenEmbedder``. Concretely, it is a dictionary of namespaces which have been indexed
-            to their corresponding tensors. At its most basic, using a SingleIdTokenIndexer this is:
-            {"tokens": Tensor(batch_size, sequence_length)}. This dictionary will have as many
-            items as you have used token indexers in the ``TextField`` representing your sequence.
-            This dictionary is designed to be passed directly to a ``TokenEmbedder``, which knows
-            how to combine different word representations into a single one per token in your input.
+            ``TextFieldEmbedder``. Concretely, it is a dictionary of namespaces which have been
+            indexed to their corresponding tensors. At its most basic, using a SingleIdTokenIndexer
+            this is: {"tokens": Tensor(batch_size, sequence_length)}. This dictionary will have as
```

Comment:
Can this now reference explicitly that the keys in this dict are the keys of the `token_indexers`.